src/storage/metrics.rs

```diff
@@ -105,4 +105,12 @@ lazy_static! {
             "tikv_storage_gc_skipped_counter",
             "Total number of gc command skipped owing to optimization"
         ).unwrap();
+
+    pub static ref BATCH_COMMANDS: HistogramVec =
+        register_histogram_vec!(
+            "tikv_storage_batch_commands_total",
+            "Bucketed histogram of total number of a batch of commands",
+            &["type"],
+            linear_buckets(0.0, 2.0, 20).unwrap()
```

Comment:
Actually, it rarely exceed 20 during my benchmarks, although the batch size is 256.